docs/posts/pydata-performance/dask_impl.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import logging
+
+import dask.config
+import dask.dataframe as pd
```

Comment:
The first thing I would recommend is to use dask_expr, which contains a re-implementation of dask.dataframe that has a query planning layer

Reply:
`dask_expr` is a bit too new AFAIK and not in common use, that's why I didn't use it.

Reply:
FWIW I am expecting it to be the default in dask before too long

Reply:
Also, if it reassures you, you can use dask-expr and still win.  DuckDB generally does much better than dask-expr on single-machines.  In your shoes I wouldn't worry too much about comparing against the newer system.

Reply:
> Also, if it reassures you, you can use dask-expr and still win.

Not super concerned about winning :)

If it turned out that Dask beat DuckDB on memory usage and latency then it would a super interesting data point for us.

The post is mostly to showcase Ibis as a viable performant alternative when dealing with "medium" data.

Noted about dask-expr!

Reply:
@mrocklin Where are the rest of the TPC-H queries in that video? I only see 1-7.

Reply:
We copied the existing implementations from the Polars repository and they only had 1-7

We will add more at some point, but that's not a super high priority at the moment